### DISCUSSION TITLE
Tabulated paragraph but not using table

### DISCUSSION THREAD

**Discussion by yoonghm** [quarto-dev/quarto-cli#6601] [Q&A] [answered]
### Description

I have the following table:

```
**Elements** | **Properties**          |                                          |
-------------|-------------------------|:-----------------------------------------|:--
Spring       | $\rightarrow$ Stiffness | Potential energy (PE): $\frac{1}{2}kx^2$ | } Energy-storage element
Mass         | $\rightarrow$ Inertia   | Kinetic energy (KE): $\frac{1}{2}mv^2$   | } Energy-storage element
Damper       | $\rightarrow$ Damping   |                                          | } Energy-dissipating element

: {tbl-colwidths="[10,15,30,30]"}
```

Is there a way to display this table using tab positions, as commonly used in Microsoft Words?

**Comment by mcanouil** [comment]
I have to admit that I don't understand the question.
Could you provide a complete Quarto document and possibly add something to show what you want to achieve in which format?

**Comment by yoonghm** [comment]
I would like to find out if I could format the following using Quarto:
![image](https://github.com/quarto-dev/quarto-cli/assets/4215991/1209e53f-94e9-4801-a8a8-b525fcacc15a)

**Reply by rgaiacs** [reply]
What is the format that you are targeting? PDF only? HTML only? PDF and HTML?

**Reply by yoonghm** [reply]
I would like to generate PDF file.

**Reply by mcanouil** [reply]
If you have no intention of generating other format you could use raw LaTeX directly.

````qmd
```{=latex}
<code>
```
````

(I am away from a keyboard right now, so can't share more/alternative solution)

**Reply by mcanouil** [reply, marked as answer]
If you have no intention of generating other format you could use raw LaTeX directly.

````qmd
```{=latex}
<code>
```
````

(I am away from a keyboard right now, so can't share more/alternative solution)

Also note that providing a complete Quarto document (as requested earlier) helps everyone help you (and avoid many back and forth for question such as the format).

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by yoonghm** [reply]
I will use latex then. Thanks.
